load-tests/README.md

```diff
@@ -0,0 +1,28 @@
+# Locust based load tests for RH-Che/CReW
+
+### Prepare environment
+
+It is recommended to do python-pip installations in a separate python virtual-env
```

Comment:
Please go through this file - all senteces, which starts with large number, should end with a dot. ;)